src/Simulation/Type2Core/Microsoft.Quantum.Type2.Core.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.12.20072031">
+
+  <Import Project="..\Common\AssemblyCommon.props" />
+  <Import Project="..\Common\DebugSymbols.props" />
+
+  <Import Project="..\TargetDefinitions\TargetPackages\Type2.Package.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <QsharpDocsGeneration>true</QsharpDocsGeneration>
+    <CsharpGeneration>false</CsharpGeneration> <!-- we will provide our own -->
+    <IncludeQsharpCorePackages>false</IncludeQsharpCorePackages>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <Description>Type2 Targeting support for the Q# programming language.</Description>
```

Comment:
@alan-geller This is the project that I imagine would participate in your new rewrite step, since it would be responsible for establishing the base classes that need to be implemented for Type2 targeting compatibility.

Reply:
Actually, thinking about it more, we would eventually want every targeting package, including QsharpCore, to use that rewrite step as the means for hooking into the simulation and/or translation infrastructure. This is just the first non-default target.